### DISCUSSION TITLE
Data TTL not effective

### DISCUSSION THREAD

**Discussion by yi121013215** [apache/skywalking#8668] [Q&A]
I modify the recordDataTTL to 2 from 3, metricsDataTTL to from 7, but the es6 data do not delete.
https://user-images.githubusercontent.com/10719426/158087424-0410d0bd-2882-46ff-9b89-8dbcf6cadff4.PNG
https://user-images.githubusercontent.com/10719426/158087489-6e16c70b-8c8c-4ef6-81f3-a77bf0166e8d.PNG
https://user-images.githubusercontent.com/10719426/158087617-a71223f1-06ae-493b-bebd-f696043f9e47.PNG

oap version：8.9.1
es version : 6.8.18

I expect that the disk will not grow all the time and will eventually reach a peak.

**Comment by wu-sheng** [comment]
You should check the status rather than the disk. We are just deleting the indices, how the disk volume acts are ElasticSearch's responsibility. And you should be careful about the timezone, deleting only happens according to OAP's Java time and timezone.
